examples/js/controls/TransformControls.js

```diff
@@ -161,36 +165,45 @@
 					child.position.set( 0, 0, 0 );
 					child.rotation.set( 0, 0, 0 );
 					child.scale.set( 1, 1, 1 );
+
 				}
+
 			});
 
 		};
 
 		this.hide = function () {
-			this.traverse(function( child ) {
```

Comment:
And, yes, to answer your question. You can't easily use these Gizmo classes individually. No methods for event handlers, for instance.